src/sphinx/admin/upgrade.rst

```diff
@@ -0,0 +1,61 @@
+Upgrade
+=======
+
+Migration to schemas.metadata.geo.ca
+------------------------------------
+
+The published location of the HNAP schema has changed from
+`nap.geogratis.gc.ca <http://nap.geogratis.gc.ca/metadata/tools/schemas/metadata/can-cgsb-171.100-2009-a/>`__ to
+`schemas.metadata.geo.ca <https://schemas.metadata.geo.ca/2009/>`__.
+
+The location of the HNAP registry used for code list definitions has also changed:
+
+* ``http://nap.geogratis.gc.ca/metadata/register/napMetadataRegister.xml``
+  | ``https://schemas.metadata.geo.ca/register/napMetadataRegister.xml``
+* ``http://nap.geogratis.gc.ca/metadata/register/registerItemClasses-eng.html``
+  | ``https://schemas.metadata.geo.ca/register/registerItemClasses-eng.html``
+
+Individual records can be upgraded by saving and loading in the editor:
+
+1. Open the record in the editor.
+2. Select :command:`Save and Close`.
+
+To update all records:
+
+1. Navigate to :menuselection:`Admin console > Tools > Batch process`.
+2. Select records to process, or choose ``All``.
+3. Use :guilabel:`Configure a process` to select the process ``URL replacer``
```

Comment:
This process is not going to replace all the urls, as only is handling `gmd:URL` elements and `xlink:href` attributes:

https://github.com/geonetwork/core-geonetwork/blob/a1e7663a50e46f916c603707013309a3dd0eca4f/schemas/iso19139/src/main/plugin/iso19139/process/url-host-relocator.xsl#L57-L72

Reply:
Do we need to extend url-host-relocator.xsl? Or should we remove this section and rely on the SQL procedure?